src/translation/swedish.c

```diff
@@ -749,6 +750,8 @@ static translation_string all_strings[] = {
     {TR_CITY_WARNING_DATA_COPY_NOT_SUPPORTED, "Kan inte kopiera från denna byggnad. Can't copy from this building" },
     {TR_CITY_WARNING_DATA_PASTE_SUCCESS, "Klistrade in inställningar" },
     {TR_CITY_WARNING_DATA_PASTE_NOT_SUPPORTED, "Kan ej klistra in" },
+    {TR_CITY_WARNING_DATA_MOTHBALL_ON, "Building is now mothballed" },
```

Comment:
You don't have to add the English translation strings to other languages when you don't know how to translate. The game already does that automatically and as a bonus it warns which strings haven't been translated yet.